src/config.rs

```diff
@@ -177,6 +183,12 @@ impl Config {
             )));
         }
 
+        // default MaxCommittedSizePerReady to MaxSizePerMsg because they were
+        // previously the same parameter.
+        if self.max_committed_size_per_ready == 0 {
+            self.max_committed_size_per_ready = self.max_size_per_msg
```

Comment:
However, it may not be backward compatible. I suggest to just leave it to `NO_LIMIT` instead.